Add propertyUpdated on fromStringArray method

tiaguinhor opened the pull request:
https://github.com/calcinai/xero-php/issues/786

tiaguinhor committed:
Add propertyUpdated on fromStringArray method

https://github.com/calcinai/xero-php/issues/786

calcinai commented:
@tiaguinhor thanks for the PR.  Just thinking about this, I'm worried it may change some behaviour if this method is being used to load cached data within an application (or something similar).

Do you think adding another parameter to the function would be an acceptable solution to maintain backward compatibility?

tiaguinhor commented:
Hey @calcinai, I think the second argument was already designed on that purpose. But yes, a third argument would be a solution too. I see on the repository we have some TDD, none of them tests those behaviours? Btw, I didn't run them. Cheers, mate.

Commit:
Attachment Triat Quote Model

tiaguinhor committed:
Merge pull request #1 from tiaguinhor/add_quotes_attachments-1

Attachment Triat Quote Model

calcinai commented:
@tiaguinhor adding that propertyUpdated method there will force all requests to contain the entire object, and I'm not sure that is desirable.

tiaguinhor committed:
Update Organisation.php

Add Class attribute https://developer.xero.com/documentation/api/accounting/organisation/#get-organisation

tiaguinhor committed:
Update Organisation.php

Add getClass method
